src/lib/components/Tabs/_styles.scss

```diff
@@ -88,6 +88,7 @@ web-tabs {
     display: flex;
     flex: 1;
     flex-direction: column;
+    overflow-y: scroll;
```

Comment:
@khempenius has a PR which does this too. Why not `auto`, which will only optionally show the scrollbar?